src/packet_analysis/protocol/ip/IP.cc

```diff
@@ -62,7 +62,8 @@ bool IPAnalyzer::AnalyzePacket(size_t len, const uint8_t* data, Packet* packet)
 			return false;
 			}
 
-		ip_hdr = std::make_shared<IP_Hdr>((const struct ip6_hdr*)data, false, len);
+		packet->ip_hdr = std::make_shared<IP_Hdr>((const struct ip6_hdr*)data, false,
```

Comment:
This appears to be the change that broke the btests on the builds. Probably just an issue with the rebase.

```suggestion
		ip_hdr = std::make_shared<IP_Hdr>((const struct ip6_hdr*)data, false,
```